htdocs/takepos/admin/terminal.php

```diff
@@ -148,6 +150,11 @@
 print '<td>'.$langs->trans("Parameters").'</td><td>'.$langs->trans("Value").'</td>';
 print "</tr>\n";
 
+print '<tr class="oddeven"><td class="fieldrequired">'.$langs->trans("TerminalNameDesc").'</td>';
+print '<td>';
+print '<input type="text" name="terminalname'.$terminal.'" value="'.(! empty($conf->global->{"TAKEPOS_TERMINAL_NAME_".$terminal}) ? $conf->global->{"TAKEPOS_TERMINAL_NAME_".$terminal} : $langs->trans("TerminalName", $terminal)).'" >';
```

Comment:
Syntax  "->{"
must be avoid.
You can use instead   getDolGlobalString("TAKEPOS_TERMINAL_NAME_".$terminal, $langs->trans("TerminalName", $terminal));